pages/about/groups/agwg.md

```diff
@@ -33,67 +34,80 @@ Some information on this page is also shown on, and may be more current in, the
 {% include_cached toc.html type="end" %}
 {:/}
 
-## Announcements
+## Admin home page
 
-No announcements at the moment.
+The AG Working Group’s administrators manage the group’s information on [Accessibility Guidelines Working Group — W3C Groups](https://www.w3.org/groups/wg/ag/).
 
-## Current Work
+## Charter
 
-The Working Group maintains a comprehensive list of [publications and current timelines](https://www.w3.org/WAI/GL/wiki/Timelines).
+The group is chartered until October 2025.
 
-The [Project Plan](https://www.w3.org/WAI/GL/wiki/Timelines) details intended timeline and milestones for this work. A [list of publications](https://www.w3.org/TR/tr-groups-all#tr_Web_Content_Accessibility_Guidelines_Working_Group) on the W3C Technical Reports page includes completed deliverables that are no longer worked on by the Working Group.
+To learn about the group’s current focus, scope and deliverables, see the [AG Working Group Charter](https://www.w3.org/WAI/GL/charter).
+
+## Current work
+
+The work of the AG Working Group follows the [W3C Process](https://www.w3.org/2023/Process-20231103/). 
+
+This work is partly funded by the [WAI Core 2015 Project](https://www.w3.org/WAI/about/projects/wai-core-2015/). The work of the AG Working Group does not necessarily reflect the views or policies of the funders.
+
+For details of the current work, see the [AG Working Group wiki](https://www.w3.org/WAI/GL/wiki/Main_Page).
 
 ### WCAG 3
 
-Work on W3C Accessibility Guidelines 3 takes place in many task forces and subgroups of the Working Group. Information about projects and timelines is available on the [WCAG 3 Project Plan](https://github.com/w3c/silver/wiki).
+Work on updating WCAG 3 takes place in many task forces and subgroups of the AG Working Group. For information about the WCAG 3 timelines and publication plan, see [WCAG 3 Timeline](https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline).
```

Comment:
This is not work on 'updating' WCAG 3. That would imply that there is something to update. This should be 'Work on WCAG 3...'

Reply:
Good point. I'd like to use a verb to make this sentence clearer. Would 'developing' work for you?